dev/com.ibm.ws.security.openidconnect.server/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -168,6 +168,12 @@ requireRequestUriRegistration.desc=Indicate by true or false whether require req
 customClaims=Extra claims of ID token
 customClaims.desc=The extra claims to be put in the payloads of the ID Token, in addition to the default realmName, uniqueSecurityName and groupIds claims.
 
+thirdPartyIDTokenClaims=Extra claims from a third-party ID Token
+thirdPartyIDTokenClaims.desc=The extra claims from a third-party ID Token to be put in the payload of the ID Token.
```

Comment:
this message is a little unclear - "ID Token" could be confused with "third-party ID Token". Would it be correct to say:

The extra claims from a third-party ID Token to be put in the payload of the OpenID Connect ID Token.